app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -64,7 +64,7 @@ description: |
   salt_enabled = host_param('salt_master') ? true : false
   chef_enabled = @host.respond_to?(:chef_proxy) && @host.chef_proxy
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
-  use_ntp = host_param_true?('use-ntp', (is_fedora && os_major < 16) || (rhel_compatible && os_major <= 7))
+  use_ntp = host_param_true?('use-ntp', (is_fedora && os_major < 16) || (rhel_compatible && os_major <= 6))
```

Comment:
Chrony is also available for RHEL 6 [0], however I wasn't able to find any docs about if it should be used rather than the `ntp` package

[0] https://access.redhat.com/downloads/content/rhel---6/x86_64/168/chrony/2.1.1-2.el6_8/x86_64/fd431d51/package